tests/topotests/lib/common_config.py

```diff
@@ -1506,14 +1506,12 @@ def generate_ips(network, no_of_ips):
                 return ipaddress_list
             start_ip = ipaddress.IPv4Address(frr_unicode(start_ip))
             step = 2 ** (32 - mask)
-        elif addr_type == "ipv6":
+        if addr_type == "ipv6":
             if start_ip == "0::0" and mask == 0 and no_of_ips == 1:
                 ipaddress_list.append("{}/{}".format(start_ip, mask))
                 return ipaddress_list
             start_ip = ipaddress.IPv6Address(frr_unicode(start_ip))
             step = 2 ** (128 - mask)
-        else:
```

Comment:
What's the purpose of this change - why not return an error if the addr_type is neither v4 nor v6? the result of the change is that ... there will be an exception? why not just raise one, if that's the right behavior for invalid input?